etc/adminhtml/system.xml

```diff
@@ -50,6 +50,13 @@
                         <field id="is_fastlane_enabled">1</field>
                     </depends>
                 </field>
+                <field id="wallet_payment_title" translate="label" type="text" sortOrder="40" showInDefault="1" showInWebsite="1">
+                    <label>Digital Wallets Title</label>
+                    <comment><![CDATA[Digital Wallets title that will be displayed on the checkout page.]]></comment>
```

Comment:
Has @jennajaco approved this copy?

Reply:
Updated the description based on feedback from @jennajaco